pkg/controller/volume/persistentvolume/binder_test.go

```diff
@@ -45,7 +45,7 @@ func TestSync(t *testing.T) {
 		{
 			// syncClaim binds to a matching unbound volume.
 			"1-1 - successful bind",
-			newVolumeArray("volume1-1", "1Gi", "", "", v1.VolumePending, v1.PersistentVolumeReclaimRetain, classEmpty),
+			newVolumeArray("volume1-1", "1Gi", "", "", v1.VolumeAvailable, v1.PersistentVolumeReclaimRetain, classEmpty),
```

Comment:
I see you change all tests to use `VolumeAvailable`. Can you please add a test with `VolumePending` to see it's really skipped?

Reply:
This has been added in this new commit: https://github.com/kubernetes/kubernetes/commit/1f8b4b1e1444529d6a536180d48c775e0aac7109

Reply:
I meant a test in `binder_test.go`, as it goes through whole syncClaim. Just in case.

Reply:
Added a test case in `TestSync`, please check https://github.com/kubernetes/kubernetes/pull/68673/commits/0c967e4720698a5567094deee5117f472a3a3d60#diff-1e6b1611f17f082fe07eac7bc9116f73R227.